build.gradle

```diff
@@ -303,26 +306,58 @@ task coverage {
 task checkStyle {
 }
 
+ext {
+    guiTests = 'address/guitests/*'
+    guiUnitTests = 'address/guiunittests/*'
+    unitTests = 'address/unittests/*'
```

Comment:
Is this fix easily extended to the scenario where we dismantle the unittests package to equalize test/functional packages?